xds/src/test/java/io/grpc/xds/XdsTestControlPlaneService.java

```diff
@@ -75,25 +76,32 @@ public void uncaughtException(Thread t, Throwable e) {
       "type.googleapis.com/envoy.config.endpoint.v3.ClusterLoadAssignment";
 
   private final Map<String, HashMap<String, Message>> xdsResources = new HashMap<>();
-  private ImmutableMap<String, HashMap<StreamObserver<DiscoveryResponse>, Set<String>>> subscribers
+  private ImmutableMap<String, Map<StreamObserver<DiscoveryResponse>, Set<String>>> subscribers
       = ImmutableMap.of(
-          ADS_TYPE_URL_LDS, new HashMap<StreamObserver<DiscoveryResponse>, Set<String>>(),
-          ADS_TYPE_URL_RDS, new HashMap<StreamObserver<DiscoveryResponse>, Set<String>>(),
-          ADS_TYPE_URL_CDS, new HashMap<StreamObserver<DiscoveryResponse>, Set<String>>(),
-          ADS_TYPE_URL_EDS, new HashMap<StreamObserver<DiscoveryResponse>, Set<String>>()
-          );
+      ADS_TYPE_URL_LDS,
+      Collections.synchronizedMap(new HashMap<StreamObserver<DiscoveryResponse>, Set<String>>()),
```

Comment:
Why not just use ConcurrentHashMap rather than doing the Collection.synchronizedMap?

Reply:
Just used to using that, but I switched over since it is more concise.